pkg/ebpf/netlink/conntracker.go

```diff
@@ -256,6 +258,12 @@ func (ctr *realConntracker) register(c ct.Conn) int {
 	return 0
 }
 
+func (ctr *realConntracker) logExceededSize() {
+	if ctr.exceededSizeLogLimit.ShouldLog() {
+		log.Warnf("exceeded maximum conntrack state size: %d entries. You may need to increase system_probe_config.max_tracked_connections (will log first ten times, and then once every 10 minutes)", ctr.maxStateSize)
```

Comment:
nit: Maybe the `ten` here should be `firstN` as a variable?

Reply:
`firstN` changes value over as  LogLimit.ShouldLog gets called.